src/rpc/evo.cpp

```diff
@@ -466,7 +486,8 @@ static UniValue protx_register_wrapper(const JSONRPCRequest& request,
     if (specific_legacy_bls_scheme)
         ptx.nVersion = CProRegTx::LEGACY_BLS_VERSION;
     else
-        ptx.nVersion = CProRegTx::GetVersion(llmq::utils::IsV19Active(::ChainActive().Tip()));
+        ptx.nVersion = CProRegTx::GetVersion(isV19active);
```

Comment:
should be using brackets

Reply:
@PastaPastaPasta Why? `CProRegTx::GetVersion` is a function

Reply:
the `if.. else..` should be using brackets so it's more clear what is inside the if and else scope. I know you're kinda not touching it, but you also introduced the `specific_legacy_bls_scheme` check so I can still yell at you for it :P